ic-agent/src/agent/http_transport.rs

```diff
@@ -263,10 +263,11 @@ mod test {
         }
 
         test("https://ic0.app", "https://ic0.app/api/v2/");
+        test("https://IC0.app", "https://ic0.app/api/v2/");
         test("https://foo.ic0.app", "https://ic0.app/api/v2/");
-        test("https://foo.bar.ic0.app", "https://ic0.app/api/v2/");
-        test("https://ic0.app/foo/", "https://ic0.app/foo/api/v2/");
-        test("https://foo.ic0.app/foo/", "https://ic0.app/foo/api/v2/");
+        test("https://foo.IC0.app", "https://ic0.app/api/v2/");
+        test("https://foo.Ic0.app", "https://ic0.app/api/v2/");
+        test("https://foo.iC0.app", "https://ic0.app/api/v2/");
```

Comment:
Could you add rather than replace these tests?  Two of the replaced tests had output that differed from the others (now every test output ends in `/ic?.app/api/v2/`)

Reply:
Whoops. Meant to just add test lines. Copy-paste error.